### ISSUE TITLE
[Question]: nes Hardware Light Gun

### ISSUE THREAD

**Issue by soundsnow** [Rakashazi/emu-ex-plus-alpha#639]
### Describe your question

[LCD/LED TV Light Gun?](https://github.com/libretro/libretro-fceumm/pull/526)

**Comment by DigiDwrf** [comment]
> ### Describe your question
> [LCD/LED TV Light Gun?](https://github.com/libretro/libretro-fceumm/pull/526)

Indeed, now you would be able to use a physical lightgun/zapper mapped inside Retroarch's lightgun device. "Trigger" for trigger, and "AUX_A" for light detection. Signal comes inverted by default, but this option can be disabled to match any configuration you would need for your hardware.
